resources/JsonApi.yaml

```diff
@@ -0,0 +1,276 @@
+            meta:
+              $ref: '#/components/schemas/Meta'
+
+    Error:
+      type: object
+      required:
+        - id
+        - status
+        - code
+        - title
+        - details
+      properties:
+        id:
+          type: string
+          description: A unique identifier for this particular occurrence of the problem.
+        status:
+          type: string
+          description: The HTTP status code applicable to this problem, expressed as a string value.
+        code:
+          type: string
+          description: An application-specific error code, expressed as a string value.
+        title:
+          type: string
+          description: A short, human-readable summary of the problem that SHOULD NOT change from occurrence to occurrence of the problem.
+        details:
+          type: string
+          description: A human-readable explanation specific to this occurrence of the problem.
+      example:
+        id: "6de7da3e-8877-4f2b-a670-16e18e5d79a0"
+        status: "400"
+        code: "31201"
+        title: "Bad Request"
+        detail: "Parameter 'Foo' is invalid"
+
+  ####################
+  # Common parameters
+  ####################
+  parameters:
+    idempotencyKey:
+      in: header
+      name: Idempotency-Key
+      required: true
+      description: "Client unique generated value for server to recognize subsequent retries"
+      schema:
+        type: string
+      example: "518d5812-b8e8-43a4-b3b0-8f9c5cdc31b9"
+    pageNumber:
+      in: query
+      name: page[number]
+      required: false
+      example: 1
+      schema:
+        type: number
+        minimum: 1
+      description: Page number if the results have more items than items per page. Default = 1.
+    offsetParam:
+      in: query
+      name: offset
+      schema:
+        type: integer
+        minimum: 0
+      description: The number of items to skip before returning the results
+    limitParam:
+      in: query
+      name: limit
+      schema:
+        type: integer
+        format: int32
+        minimum: 1
+        maximum: 100
+        default: 20
+      description: The number of items to return
+    language:
+      name: language
+      in: query
+      required: false
+      description: "The PK of the language to use, which will be a 3 character ISO 639-2 string (e.g.: fin). If this is omitted the system defined default will be used. If the requested language is not available then the system will return translations in order of the system defined preferences"
+      schema:
+        type: string
+
+  #######################
+  # Common request bodies
+  #######################
+  requestBodies:
+    NewItem:
+      description: A JSON object containing item data
+      required: true
+      content:
+        application/vnd.api+json:
+          schema:
+            type: object
+
+  ####################
+  # Common responses
+  ####################
+  responses:
```

Comment:
There's actually a couple more in [Kinescope](https://github.com/24i/prd-kinescope/blob/master/resources/openapi/JsonApi.yaml)
Let's add those too.

Reply:
I also added a few additional ones from importer project.